pwndbg/color.py

```diff
@@ -53,7 +53,7 @@ def get(address, text = None):
     if text is None and isinstance(address, (long, int)) and address > 255:
         text = hex(int(address))
     if text is None:
-        text = int(address)
+        text = str(int(address))
```

Comment:
I think not using `str` here was a bug.

The `address > 255` in the first if statement seems somewhat arbitrary. In my case, the `last_remainder` chunk starts of as `0` instead of `0x0` because of this.